src/extension.ts

```diff
@@ -75,7 +75,7 @@ export async function activate(context: vscode.ExtensionContext) {
     // Start language server
     startLanguageServer();
 
-    if (vscode.workspace.getConfiguration('julia').get<boolean>('enableTelemetry')===null) {
+    if (vscode.workspace.getConfiguration('julia').get<boolean>('enableTelemetry') === null) {

```

Comment:
There are a lot of these formatting changes in this PR, any chance you could get rid of those? Makes it easier to check what really is changed. If not, no worries.

Reply:
Reverted. I just subconsciously pressed Format Document lol